apiserver/params/leadership.go

```diff
@@ -40,11 +40,11 @@ type ReleaseLeadershipBulkParams struct {
 // leadership claim.
 type ReleaseLeadershipParams struct {
```

Comment:
I don't think this is used anywhere (and there's not a way to do it in the underlying store interface atm) so maybe just remove it?